wagtail/contrib/search_promotions/models.py

```diff
@@ -50,12 +69,28 @@ def garbage_collect(cls):
 
     @classmethod
     def get(cls, query_string):
+        """
+        Gets or creates object matching query_string.
+
+        Args:
+           - query_string(str): Query text.
+
+        Returns:
+           - First Query object matching query_string.
+        """
```

Comment:
```suggestion
```

This docstring can be removed, it's longer than the actual method. In this case it's very easy to read this code and know what's happening.